morpheus-graphql-client/src/Data/Morpheus/Client/Declare/Aeson.hs

```diff
@@ -132,18 +132,19 @@ deriveFromJSON ClientTypeDefinition {clientCons = [], clientTypeName} =
     $ "Type "
       <> msg (typename clientTypeName)
       <> " Should Have at least one Constructor"
+deriveFromJSON ClientTypeDefinition {clientTypeName, clientCons}
+  | isEnum clientCons =
```

Comment:
i think it would be better to use  `clientKind` property for it. would achieve more robust deriving.

```hs
ClientTypeDefinition {clientTypeName, clientCons, clientKind = KindEnum} = 
```

Reply:
I'll look into using that

Reply:
Done.